src/helpers/wpcom-functions.php

```diff
@@ -2,6 +2,51 @@
 
 namespace Team51\Helper;
 
+/**
```

Comment:
What made you decide to add these as helpers rather than include this functionality in the main command? The general likelihood that we'll use them again for future CLI efforts?

Reply:
> The general likelihood that we'll use them again for future CLI efforts?

Yes! If we use the WPCOM API at all, these are the most likely endpoints that we'll use ... and, overall, it just keeps the code inside the command cleaner.

Personally, I think that this is verbose enough: ![0LrSpc.png](https://user-images.githubusercontent.com/36231502/227214790-fbdb491d-8ab9-434f-a792-db40b45c3a4f.png)

If that `if` statement is triggered when it shouldn't, then I know to look into `get_wpcom_jetpack_sites` and nowhere else. My current command doesn't matter, and I can forget about it while looking into it.

Last but not least, I could write down the differences between the different endpoints in the form of comments (e.g., why use `get_wpcom_sites` instead of `get_wpcom_jetpack_sites`).